### ISSUE TITLE
the detection box is too large

### ISSUE THREAD

**Issue by jacobssy** [experiencor/basic-yolo-keras#210]
@experiencor Hi, I trained a model , loss, recall looks good, but in test step, the detection box is too large 
![image](https://user-images.githubusercontent.com/14237914/37895497-05a2b4b0-3114-11e8-8c24-2ffde698346b.png)
I have adjust the NMS_THRESHOLD, but it did not  improve a lot , what should I do when I trained that, or some change in prediction 
Thanks! :)

**Comment by experiencor** [comment]
@jacobssy This is strange if the recall looks good. What's your final recall? Regarding the size of the box, you may increase the size accuracy by increasing ```coord_scale```.

**Comment by jacobssy** [comment]
@experiencor , recall nearly rises up to 98%-99%,  I will retrain it  by increasing coord_scale. :)

**Comment by jacobssy** [comment]
@experiencor Hi, I have retrained it for a epoch 
![image](https://user-images.githubusercontent.com/14237914/37897751-889001ac-3119-11e8-9f9f-8d770091433e.png)
it is weird, the recall and loss look good(I have enlarge the value of coord_scale)But the result is not good ,the detection box still very large

**Comment by experiencor** [comment]
@jacobssy Have you set warmup_epoches to 0?

**Comment by jacobssy** [comment]
@experiencor yes,I did , it is 0

**Comment by jacobssy** [comment]
@experiencor Hi, sry for calling you again, I retrained the model, I increasing the **coord_scale** to 30,and object_scale to 10, trained for 60 epoch,the loss is good,recall also good, and I test the model,the result is also strange,the detection box is too large
![image](https://user-images.githubusercontent.com/14237914/37943809-c05ccb5a-31ab-11e8-921c-58ca7cf9b045.png)
is there any possible relationship with grid_w,grid_h value? 13x13 maybe not good for the dataset? because the object is too small in the whole picture?

**Comment by experiencor** [comment]
@jacobssy I doubt that your dataset has some scale problem. May you check your labels again? I cannot think of any reason that makes recall high but so bad prediction accuracy.

**Comment by jacobssy** [comment]
@experiencor Hi,I have retrained that by setting input_size = 480 and grid_h = 15, and test result is good,the detection box is fine. But I am still confused about last result.. I will check that. Thanks for your reply
